projects/plugins/jetpack/jetpack-user-content-tracking-redirects.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/**
+ * User Content Tracking Redirects
```

Comment:
Since this is useful in emails sent via the Subscriptions feature, could we move this code to `modules/subscriptions.php`, where it will only be triggered when the Subscriptions module is active?

Reply:
One caveat I can foresee is if the subscriptions module is disabled, user content links on emails that has already been sent out will stop working, but maybe that's a feature, not a bug. :)

Reply:
It will now no longer redirect when subscription module is disabled.